ui/src/overridden/backoffice/PatronMetadata/PatronMetadata.js

```diff
@@ -0,0 +1,45 @@
+import {
+  ScrollingMenuItem,
+  MetadataTable,
+} from '@inveniosoftware/react-invenio-app-ils';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Grid, Header, Segment } from 'semantic-ui-react';
+
+export const PatronMetadata = ({ patron, ...props }) => {
+  const leftTable = [
+    { name: 'Name', value: patron.metadata.name },
+    { name: 'Email', value: patron.metadata.email },
+  ];
+  const rightTable = [
+    { name: 'Person ID', value: patron.metadata.person_id },
```

Comment:
Let me know if there are any other CERN fields we would like to display, or if we don't want to have a whole segment but move them to the header.